vrx_gazebo/include/vrx_gazebo/follow_plugin.hh

```diff
@@ -49,6 +51,11 @@ namespace gazebo
 ///              the model should navigate through. This block should contain at
 ///              least one of these blocks:
 ///                <waypoint>: This block should contain the X, Y of a waypoint.
+/// <line>: Element that indicates the model should travel in "line" mode.
+///         The block should contain the relative direction from the initial
+///         position in which the vehicle should move, specified in the world
+///         frame.
```

Comment:
Could you specify the units? Could you document the meaning of `<distance>` and its units in this context?